bitnami/mariadb-galera/templates/statefulset.yaml

```diff
@@ -242,6 +242,8 @@ spec:
           image: {{ template "mariadb-galera.metrics.image" . }}
           imagePullPolicy: {{ .Values.metrics.image.pullPolicy | quote }}
           env:
+            - name: MARIADB_METRICS_EXTRA_FLAGS
+              value: {{ default "" .Values.metrics.extraFlags | quote }}
```

Comment:
nit: Do we really need the environment variable? I mean, there's no harm in having it, just not sure if it's necessary. Did you have any thoughts around it?

Reply:
I choose this over putting it directly in the command for some better visibility (eg if in the future anyone decides to put in default flags). It was also a bit annoying putting it in directly because gotemplates kept wanting to append a newline after when the value is nil, so the only way to get around it was to have if statements, which was a bit ugly for me.